savant/client/frame_source/jpeg.py

```diff
@@ -26,6 +27,12 @@ class JpegSource(FrameSource):
 
     :param source_id: Source ID.
     :param filepath: Path to a JPEG file.
+    :param pts: Frame presentation timestamp.
+    :param framerate: Framerate (numerator, denominator).
+    :param source_id_add_size_suffix: Whether to add a suffix
```

Comment:
This parameter (source_id_add_size_suffix) looks strange. Could you explain the reason for it?

Reply:
Removed it.